spyder/dependencies.py

```diff
@@ -11,7 +11,93 @@
 
 # Local imports
 from spyder.utils import programs
-
+from spyder.config.base import _
+CLOUDPICKLE_REQVER = '>=0.5.0'
+PYGMENTS_REQVER = '>=2.0'
+QTCONSOLE_REQVER = '>=4.5.0'
+NBCONVERT_REQVER = '>=4.0'
+SPHINX_REQVER = '>=0.6.6'
+PYLINT_REQVER = '>=0.25'
+PSUTIL_REQVER = '>=0.3'
+QTAWESOME_REQVER = '>=0.5.7'
+QTPY_REQVER = '>=1.5.0'
+PICKLESHARE_REQVER = '>=0.4'
+PYZMQ_REQVER = '>=17'
+CHARDET_REQVER = '>=2.0.0'
+NUMPYDOC_REQVER = '>=0.6.0'
+SPYDER_KERNELS_REQVER = '>=1.4.0'
+QDARKSTYLE_REQVER = '>=2.7.0'
+ATOMICWRITES_REQVER = '>=0.0.0'
+DIFF_MATCH_PATCH_REQVER = '>=0.0.0'
+WATCHDOG_REQVER = '>=0.0.0'
+KEYRING_REQVER = '>=12.0.0'
+PEXPECT_REQVER = '>=4.4.0'
+PARAMIKO_REQVER = '>=2.4.0'
+PYXDG_REQVER = '>=0.26'
+DEPENDENCIES_BASE = [
+    {'modname': "cloudpickle", 'package_name': "cloudpickle",
+        'features': _("Serialize variables in the IPython kernel to send to "
+                      "Spyder."), 'required_version': CLOUDPICKLE_REQVER},

```

Comment:
Please reformat these entries like this

```python
{'modname': "cloudpickle",
 'package_name': "cloudpickle",
 'features': _("Serialize variables in the IPython kernel to send to Spyder.)"
 'required_version': CLOUDPICKLE_REQVER}
```

That would make them much more readable.

And don't worry about `'features'` taking more than 79 columns. Let's forget about that requirement in this file in order to improve readability.